static/openapi.json

```diff
@@ -11,17 +11,20 @@
                     "tasks"
                 ],
                 "summary": "Get Tasks",
-                "description": "Get a list of the IDs of all available labelling tasks",
+                "description": "Get a list of the IDs of all available labeling tasks",
                 "operationId": "get_tasks",
                 "responses": {
                     "200": {
                         "description": "Successful Response",
                         "content": {
                             "application/json": {
-                                "schema": {},
-                                "example": [
-                                    "ecg-qrs-classification-physiodb"
-                                ]
+                                "schema": {
+                                    "title": "Response Get Tasks",
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/components/schemas/Task"
+                                    }
+                                }
                             }
```

Comment:
```suggestion
                            },
                            "example": [
                                  "ecg-qrs-classification-physiodb",
                                  "pressure-mat-pose-estimation-slp"
                            ]
```
No must have, just for completion

Reply:
Yes, definitely don't change this file by hand. Don't know how to add it to fastapi right now, let's have a look at it together tomorrow.